workflows/flows/assemble_study_tasks/assemble_samplesheets.py

```diff
@@ -28,6 +29,72 @@
 EMG_CONFIG = settings.EMG_CONFIG
 
 
+HOST_TAXON_TO_REFERENCE_GENOME = {
+    "9031": "chicken.fna",  # Gallus gallus
+    "8030": "salmon.fna",  # Salmo salar
+    "8049": "cod.fna",  #  Gadus morhua
+    "9823": "pig.fna",  # Sus scrofa
+    "9913": "cow.fna",  # Bos taurus
+    "10090": "mouse.fna",  # Mus musculus
+    "7460": "honeybee.fna",  # Apis mellifera
+    "10116": "rat.fna",  # Rattus norvegicus
+    "8022": "rainbow_trout.fna",  # Oncorhynchus mykiss
+    "9940": "sheep.fna",  # Ovis aries
+    "3847": "soybean.fna",  # Glycine max
+    "7955": "zebrafish.fna",  #  Danio rerio
+    "Gallus gallus": "chicken.fna",
+    "Salmo salar": "salmon.fna",
+    "Gadus morhua": "cod.fna",
+    "Sus scrofa": "pig.fna",
+    "Bos taurus": "cow.fna",
+    "Mus musculus": "mouse.fna",
+    "Apis mellifera": "honeybee.fna",
+    "Rattus norvegicus": "rat.fna",
+    "Oncorhynchus mykiss": "rainbow_trout.fna",
+    "Ovis aries": "sheep.fna",
+    "Glycine max": "soybean.fna",
+    "Danio rerio": "zebrafish.fna",
+}
```

Comment:
Should it be in config?

Reply:
It's a good point Kate. My sense was that since it was a coded Enum in miassembler, it should probably be one in automation too. I think config can be **over**used for structures like this... e.g. if we put this in config, and then want to add one more reference genome, we have to duplicate this entire dict, plus one line, into a config file to override it here (or use a fancier data structure) Hmm

Reply:
Yeh, that would be nicest way. Need to think about the best process for that, though 🤔

Reply:
Created task EMG-7399 to do this properly, thanks @KateSakharova and @mberacochea .